src/main/java/com/jfb/tickets/service/TicketService.java

```diff
@@ -1,72 +1,41 @@
 package com.jfb.tickets.service;
 
+import com.jfb.tickets.DatabaseConnection;
+import com.jfb.tickets.dao.TicketDAO;
+import com.jfb.tickets.dao.UserDAO;
 import com.jfb.tickets.enums.*;
 import com.jfb.tickets.model.*;
-import com.jfb.tickets.structures.*;
+import com.jfb.tickets.structures.CustomArrayList;
 
 import java.math.BigDecimal;
+import java.sql.*;
 import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
 
 public class TicketService {
-    public static void main(String[] args) {
-
-
-        Ticket emptyTicket = new Ticket();
-        Ticket limitedTicket = new Ticket("TAURON", 378);
-        Ticket fullTicket = new Ticket("NARODOWY", 948, Instant.parse("2025-06-23T00:00:00Z"), true, StadiumSector.B, 21, new BigDecimal("123.99"));
-
-        Client client1 = new Client();
-        Admin adm = new Admin();
-
-        client1.printRole();
-        adm.printRole();
-        System.out.println();
-
-        System.out.println(client1.getTicket());
-        System.out.println(adm.checkTicket(client1.getTicket()));
```

Comment:
do not delete previous functionality. just move each task in separate method (testCollectionsTask(); testOOPTask(); etc.)